### PR TITLE
CT-3925: Add sar ir subtype to report

### PR DESCRIPTION
**Pull request by willmcb**
## Description
Amends the cases reports so as allow for SAR IRs. Adds new column to reports to show whether a SAR IR is for 'timeliness' or 'compliance'.

New column title is: 'SAR IR - timeliness or compliance'

## Self-review checklist

* [ ] (1) Quick stakeholder demo done OR
* [ ] (2) ...bug with before and after screenshots
* [x] (3) Tests passing
* [x] (4) Branch ready to be merged (not work in progress)
* [x] (5) No superfluous changes in diff
* [x] (6) No TODO's without new ticket numbers
* [x] (7) PR Prefixed with ticket number e.g. `CT-7654 ...`
* [x] (8) Data migration script is created if any of letter templates is changed

### Related JIRA tickets
[CT-3925](https://dsdmoj.atlassian.net/browse/CT-3925)

### Manual testing instructions
- you should already be able to search of SAR IR cases, so please check that you can.
- download the open cases report - you should be able to see a new column in the report that reports the SAR IR subtypes of 'timeliness' or 'compliance'

### CODE REVIEW

File: app/services/csv_exporter.rb [hunk 1]
```diff
@@ -90,7 +91,7 @@ class CSVExporter
     'in_target',
     'number_of_days_taken',
     'number_of_days_late',
-    'number_of_days_taken_after_extension',
+    'number_of_days_taken_after_extension'
```
Comment (ymao2) [inline app/services/csv_exporter.rb:94]: Not sure about this line, did you mean a new field name?

**Review by ymao2 — COMMENTED** [review]
I tried the PR, the open cases report looks,  the close report needs a little bit more work, I suggest to change the content of case_type for SAR IR rather than adding new field as it is in line with other case types and is simple. welcome more discussion,

File: app/services/csv_exporter.rb (hunk 1 above)
Comment (willmcb) [reply app/services/csv_exporter.rb:94]: No I had a new field name here and then realised that it wasn't needed - I removed the comma as it is not needed in any case 👍

**Comment by willmcb** [comment]
Closing as creating a new branch with slightly different approach

**Pull request closed without merging**